config/script.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2023 Terramate GmbH
+// SPDX-License-Identifier: MPL-2.0
+
+package config
+
+import (
+	hhcl "github.com/hashicorp/hcl/v2"
+	"github.com/terramate-io/terramate/errors"
+	"github.com/terramate-io/terramate/hcl"
+	"github.com/terramate-io/terramate/hcl/eval"
+	"github.com/zclconf/go-cty/cty"
+)
+
+// Errors for indicating invalid script schema
+const (
+	ErrScriptSchema              errors.Kind = "script config has an invalid schema"
+	ErrScriptInvalidTypeDesc     errors.Kind = "invalid type for script.description"
+	ErrScriptInvalidTypeCommand  errors.Kind = "invalid type for script.command"
+	ErrScriptInvalidTypeCommands errors.Kind = "invalid type for script.commands"
+)
+
+// EvaluatedScriptJob represents an evaluated job block
+type EvaluatedScriptJob struct {
+	Command  []string
+	Commands [][]string
+}
+
+// EvaluatedScript represents an evaluated script block
+type EvaluatedScript struct {
```

Comment:
I think the naming should align with similar types of this package, i.e.
```
// Stack represents an evaluated stack.
type Stack struct { ...

// Assert represents evaluated assert block configuration.
type Assert struct { ...
```

Reply:
Very good point. I missed that.